spec/rack/dev-mark/middleware_spec.rb

```diff
@@ -91,4 +92,14 @@
       expect(body).to eq(["response"])
     end
   end
+  context "redirection" do
+    let(:status) { 302 }
+    let(:body) { ['<html><head></head><body></body></html>'] }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.